app/controllers/mixins/ems_common_angular.rb

```diff
@@ -245,7 +245,7 @@ def set_ems_record_vars(ems, mode = nil)
 
       if ems.kind_of?(ManageIQ::Providers::Azure::CloudManager)
         ems.azure_tenant_id = params[:azure_tenant_id]
-        ems.subscription    = params[:subscription]
+        ems.subscription    = params[:subscription].blank? ? nil : params[:subscription]
```

Comment:
Ultimately, this `nil` check should be in the Armrest API.  The problem with having it here is that the ManageIQ REST API (and any other client, really) needs to know about this check.

Something to change in the Armrest API for the next version I guess, @djberg96.

Unless there's some distinct meaning behind empty string vs. `nil` for the Armrest API.